coops-ui/src/service-worker.js

```diff
@@ -0,0 +1,10 @@
+self.__precacheManifest = [].concat(self.__precacheManifest || [])
+workbox.precaching.suppressWarnings()
+workbox.precaching.precacheAndRoute(self.__precacheManifest, {})
+
+// install new service worker when ok, then reload page.
```

Comment:
See the comment I wrote in https://github.com/severinbeauvais/lear/blob/1225/coops-ui/src/service-worker.js.